### PR TITLE
Fix/design fixes

### LINKED ISSUE #138: Default spacing for content-box
content-box is being used all over the place in web-client and I have to set margins/padding/etc. by hand. We should have these in the styleguide.

Discussion: https://github.com/brainly/web-client/pull/90

### LINKED ISSUE #302: Letter spacing for uppercase text in labels/components
Apply letter-spacing for all cases where we have UPPERCASE text in labels and components.

Done for headers already.

### PR DESCRIPTION
**Pull request by mr-mig**
The fixes discussed with @JoannaWalusiak:

* paddings for containers
* component height == `rhythm(2)`
* text bit sizes + line-height
* content box margins
* etc...

Closes #302 
Closes #138

## Bonus: `uppercase` mixin

As we have been adding `letter-spacing` for uppercase texts, I have created the `uppercase($letterSpacing)` mixin with default letter-spacing == `0.03rem`

### CODE REVIEW

File: src/components/buttons/_buttons.scss [hunk 1]
```diff
@@ -1,8 +1,8 @@
-$buttonPrimaryHeight: rhythm(1 + 5 / 6);
+$buttonPrimaryHeight: rhythm(2);
```
Comment (aju) [inline src/components/buttons/_buttons.scss]: :+1:

File: src/components/buttons/_buttons.scss [hunk 2]
```diff
@@ -51,7 +51,7 @@ $iconAsButtonLightActiveColor: rgba($iconAsButtonLightColor, 0.7);
 $iconAsButtonGrayColor: $grayPrimary;
 $iconAsButtonGrayActiveColor: rgba($iconAsButtonGrayColor, 0.7);
 
-$buttonRoundAddSize: rhythm(2 + 1 / 12);
+$buttonRoundAddSize: rhythm(2);
```
Comment (aju) [inline src/components/buttons/_buttons.scss]: :+1:

File: src/components/text/_text-bits.scss [hunk 3]
```diff
@@ -61,10 +60,12 @@ $textBitLargerDesktopFontSize: fontSize(xxlarge);
   &--to-larger:not(.mint-text-bit--not-responsive) {
     @include mintBreakpoint(medium-up) {
       font-size: $textBitLargerTabletFontSize;
+      line-height: rhythm(3);
```
Comment (aju) [inline src/components/text/_text-bits.scss]: Thx, I wanted to fix it :)

File: src/components/toplayer/_toplayer.scss [hunk 4]
```diff
@@ -84,10 +85,11 @@ $includeHtml: false !default;
     }
 
     &__close {
-      @include fixText(24px, -4px);
       color: $toplayerCrossColor;
+      font-size: 32px;
```
Comment (aju) [inline src/components/toplayer/_toplayer.scss]: Maybe we could use variables for this close button sizes?

File: src/components/toplayer/_toplayer.scss [hunk 5]
```diff
@@ -84,10 +85,11 @@ $includeHtml: false !default;
...
+      height: 20px;
       font-weight: bold;
-      padding: 0 10px;
+      padding: 0 8px;
```
Comment (aju) [inline src/components/toplayer/_toplayer.scss]: Based on  layoutDefaultPadding ?

File: src/sass/_mixins.scss [hunk 6]
```diff
@@ -62,6 +62,15 @@
   // scss-lint:enable NameFormat
 }
 
+@mixin uppercase($letterSpacing: null){
```
Comment (aju) [inline src/sass/_mixins.scss]: I like the concept I don't like the name but I can't think about anything better than that :(

**Comment by aju** [comment]
Except small comments :+1: 
:sheep: :it: !! :fireworks:

**Comment by matzimowski** [comment]
:+1: :point_right: :point_left:  :older_man: :older_woman: :baby:

File: src/sass/_mixins.scss (hunk 6 above)
Comment (mr-mig) [inline src/sass/_mixins.scss]: `uppercasedText` ?

Comment (aju) [inline src/sass/_mixins.scss]: sounds better :)

File: src/components/toplayer/_toplayer.scss (hunk 4 above)
Comment (mr-mig) [inline src/components/toplayer/_toplayer.scss]: Do we need it?
I am still confused by the framework of moving things to variables.

Comment (aju) [inline src/components/toplayer/_toplayer.scss]: Skip it - we will handle all this things after moving to SVG icons

**Pull request merged**